### PR TITLE
To remove all children from parent

### PR DESCRIPTION
**Pull request by jgabrielpro**
### Description

better way to remove all children from parent

### Motivation

That will help any developer to remove all the childs of parent

### CODE REVIEW

File: files/en-us/web/api/node/removechild/index.md [hunk 1]
```diff
@@ -113,6 +113,19 @@ let garbage = top.removeChild(nested);
 garbage = top.removeChild(nested);
 ```
 
+To remove all children from an parent:
```
Comment (jpmedley) [inline files/en-us/web/api/node/removechild/index.md:116]:
```suggestion
### Removing all Children from a parent
```

Comment (jpmedley) [reply files/en-us/web/api/node/removechild/index.md:116]: This makes it consistent with the rest of the section.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/node/removechild/index.md [hunk 2]
```diff
@@ -113,6 +113,19 @@ let garbage = top.removeChild(nested);
...
+
+```js
+const parentNode = document.getElementById("parent");
+if (parentNode){
+  let child = parentNode.lastElementChild;
+  while(child){
+  parentNode.removeChild(child);
```
Comment (estelle) [inline files/en-us/web/api/node/removechild/index.md:123]:
```suggestion
    parentNode.removeChild(child);
```

File: files/en-us/web/api/node/removechild/index.md [hunk 3]
```diff
@@ -113,6 +113,19 @@ let garbage = top.removeChild(nested);
...
+  child = parentNode.lastElementChild;
```
Comment (estelle) [inline files/en-us/web/api/node/removechild/index.md:124]:
```suggestion
    child = parentNode.lastElementChild;
```

**Review by estelle — COMMENTED** [review]
code spacing.

**Comment by estelle** [comment]
Thank you for adding this function. As there is already a version of the same function in the document, closing. Thank you though. I hope you try contributing to MDN again.:

> To remove all children from an element:
```js
let element = document.getElementById("top");
while (element.firstChild) {
  element.removeChild(element.firstChild);
}
```

**Pull request closed without merging**